### PR TITLE
Updating Formatting According to WILDS Guidelines

### PR DESCRIPTION
**Pull request by tefirman**
## Description
Just performing a slight structural revamp of this repository in preparation for the WILDS WDL go-live. Nothing in terms of the WDL script's actual logic, just adjusting the formatting to align with [WILDS guidelines](https://getwilds.org/guide/wdlconfig.html). The major updates are as follows:

- Adding governance and template files according to Scott's templates
- Updating WDL script formatting according to WILDS guidelines
- Adding WOMtool validate GitHub Action to execute as a check during every pull request

### CODE REVIEW

File: ww-cell-ranger.wdl [hunk 1]
```diff
@@ -0,0 +1,95 @@
+version 1.0
+
```
Comment (sitapriyamoorthi) [inline ww-cell-ranger.wdl:2]: Do we want to add a bit of description right here about the WDL?

File: ww-cell-ranger.wdl [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+#### WORKFLOW DEFINITION
+
+workflow cell_ranger {
+  input {
+      Array[cellRangerSample] allSamples
+      String cellRangerReferenceTar
+  }
+
+  # scatter over each of the samples in the array allSamples
+  scatter ( sample in allSamples ) {
```
Comment (sitapriyamoorthi) [inline ww-cell-ranger.wdl:22]: really minor this but those spaces between the start and end of the bracket and the first and last letter respectively makes me not happy!

File: ww-cell-ranger.wdl (hunk 1 above)
Comment (tefirman) [reply ww-cell-ranger.wdl:4]: Great call, added a quick description with a link to more details from Cell Ranger.

File: ww-cell-ranger.wdl (hunk 2 above)
Comment (tefirman) [reply ww-cell-ranger.wdl]: Totally get it! Agreed! Removed the two spaces.

**Comment by tefirman** [comment]
Also added parameter_meta sections per Ted's suggestion and updating variable naming conventions according to WILDS style guidelines!

**Pull request merged**